http/http.go

```diff
@@ -237,6 +267,20 @@ func (s *Server) JSONHandler(w http.ResponseWriter, r *http.Request) *appError {
 	return nil
 }
 
+func (s *Server) OtherIPJSONHandler(w http.ResponseWriter, r *http.Request) *appError {
```

Comment:
I can't really understand the purpose of this new function and splitting `newResponse` into several, could you please clarify? Is it something to do with caching? But does anything custom need to be done? We weren't caching useragent even before, and everything else can be safely cached for a custom IP?

Reply:
I think this is cruft. removing.

Reply:
OK 🙂 In that case let's also return `newResponse` and drop the two new methods?